src/__Plugin.ino

```diff
@@ -1181,10 +1182,15 @@ byte PluginCall(byte Function, struct EventStruct *event, String& str)
                 TempEvent.sensorType = Device[DeviceIndex].VType;
                 checkRAM(F("PluginCall_s"),x);
                 START_TIMER;
-                bool retval = (Plugin_ptr[x](Function, &TempEvent, str));
+                retval = (Plugin_ptr[x](Function, &TempEvent, str));
                 STOP_TIMER_TASK(x,Function);
                 delay(0); // SMY: call delay(0) unconditionally
                 if (retval) {
+#ifdef USES_C014  // Homie 3 & 4dev Controller
+                  // check if ther is usefull information which should be sent to a controller too.
+                  // for TESTING! Not shure if here is the right place. Should perhaps be passed to all controllers in the future.
```

Comment:
Please try to minimize spelling errors in comments too

- ther => there
- shure => sure

Reply:
try my best ;)